lib/tests/streamlit/version_test.py

```diff
@@ -19,6 +19,7 @@
 
 import mock
 import requests_mock
+from packaging.version import Version as pkg_version
```

Comment:
As `pkg_version` is a class, shouldn't the name be `PkgVersion`? (not to confuse it with function)

Reply:
Nice catch